packages/core/src/components/_index.scss

```
@@ -37,6 +37,7 @@
 @import "tabs/tabs";
 @import "tag/tag";
 @import "tag-input/tag-input";
+@import "tag-input/resizeable-input";
```

missed a file name here:

```suggestion
@import "tag-input/resizable-input";
```